node/forest_libp2p/src/behaviour.rs

```diff
@@ -428,9 +432,7 @@ impl ForestBehaviour {
             ping: Ping::default(),
             identify: Identify::new(
                 "ipfs/0.1.0".into(),
-                // TODO update to include actual version
-                // https://github.com/ChainSafe/forest/issues/934
-                format!("forest-{}", "0.1.0"),
+                format!("forest-{}", *VERSION),
```

Comment:
nice! Can you add that same commit hash at the end of this as well? Something like

```suggestion
                format!("forest-{}-{}", *VERSION, *CURRENT_COMMIT),
```

Sorry if the issue was unclear. If not I can just do it in a following PR.